internal/controllers/reasons.go

```diff
@@ -0,0 +1,17 @@
+package controllers
+
+const (
+	// AddonOperator condition reasons
+	AddonOperatorReady  = "AddonOperatorReady"
+	AddonOperatorPaused = "AddonOperatorPaused"
+
+	// Addon condition reasons
+	AddonFullyReconciled      = "FullyReconciled"
+	AddonTerminating          = "Terminating"
+	AddonConfigError          = "ConfigurationError"
+	AddonPaused               = "AddonPaused"
+	AddonUnreadyCatalogSource = "UnreadyCatalogSource"
+	AddonCollidedNamespaces   = "CollidedNamespaces"
+	AddonUnreadyNamespaces    = "UnreadyNamespaces"
+	AddonUnreadyCSV           = "UnreadyCSV"
+)
```

Comment:
When we want to make these properties available for import, why not put them under `/apis` and prefix them with:
`AddonOperatorReason*` and `AddonReason*`?

We either fully document and provide consistency guarantees for these reasons and expose them as part of our `/apis` package or we keep them "hidden": only to be used by code when no dedicated condition is available.